WebAssets/mirrorsharp.js

```diff
@@ -17,7 +18,8 @@
           require('codemirror-addon-lint-fix'),
           require('codemirror/addon/hint/show-hint'),
           require('codemirror/mode/clike/clike'),
-          require('codemirror/mode/vb/vb')
+          require('codemirror/mode/vb/vb'),
+          require('codemirror/mode/php/php')

```

Comment:
Let's not include it by default (same as F#, actually I should remove VB as well).
SharpLab will include it, but otherwise it's not a mandatory dependency.